plugins/zenoh-plugin-rest/config_schema.json5

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Config",
+  "type": "object",
+  "required": [
+    "http_port"
+  ],
+  "properties": {
+    "__path__": {
+      "type": [
+        "string",
+        "null"
+      ]
+    },
+    "__required__": {
+      "type": [
+        "boolean",
+        "null"
```

Comment:
I would have expected `{"type": "boolean", "required": false}`. Is this equivalent, or is the `_required_` key now mandatory, but nullable?

Reply:
I need to check json schema standard, but at least example file without "__required__" field passes the check.
I've added additional verification of sample config file against generated schema. Seems that serde is much less strict than formal schema specification. Sometimes it seems completely wrong: schema requires `volumes` and `storages` to be vectors, but our config sample shows them as objects:
```Rust
pub struct PluginConfig {
    pub name: String,
    pub required: bool,
    pub backend_search_dirs: Option<Vec<String>>,
    pub volumes: Vec<VolumeConfig>,
    pub storages: Vec<StorageConfig>,
...
```